cypress/page-objects/app/datasets-page.ts

```diff
@@ -0,0 +1,23 @@
+import { BasePage } from "../lib/basePage";
+import { LeftPanelWidget } from "./widgets/left-panel.widget";
+import { FilterDatasetDataCy } from "../../../src/components/data-testid/dataset-page.cy";
+
+
+class DatasetPage extends BasePage {
+  readonly leftPanel: LeftPanelWidget;
+  constructor() {
+    super();
+    this.leftPanel = new LeftPanelWidget({
+      selectors: {
+        searchInput: FilterDatasetDataCy.searchInput,
+        leftPanelHeader: FilterDatasetDataCy.leftPanelHeader
+      }, labels: { leftPanelHeader: 'Datasets' }
+    })
+  }
+  getDataset(name: string) {
+    return cy.get(".flex.items-center.relative.w-full.cursor-pointer").contains(name); //name = "xl_PGP3140_wgs_NIST-4_2"
```

Comment:
Comments are redundant. Or you can use `// TODO: fix something` comment with the description.
Let's keep code clean.